internal/app/server/server.go

```diff
@@ -175,4 +177,7 @@ func registerEndpoints(ctx context.Context, g *grpc.Server, o orchestrator.Orche
 	clusterservice.RegisterClusterDiscoveryServiceServer(g, gcpv3)
 	endpointservice.RegisterEndpointDiscoveryServiceServer(g, gcpv3)
 	listenerservice.RegisterListenerDiscoveryServiceServer(g, gcpv3)
+
+	// Use https://github.com/grpc/grpc-experiments/tree/master/gdebug to debug grpc channel issues
+	service.RegisterChannelzServiceToServer(g)
```

Comment:
nit: do you know if there's any perf implications to having this turned on all the time?

Reply:
good qs..im not sure of that..since we will be actively monitoring and need tools initially while rolling out, we could make this a bootstrap setting too